### ISSUE TITLE
Remote images missing

### ISSUE THREAD

**Issue by Lutin** [SamyPesse/tv.js#7]
Remote build fails as there is no images directory under `public/remote/ressources/`

**Comment by franciscolourenco** [comment]
I got the same issue:

```
Error: ENOENT, no such file or directory '/Users/user/Downloads/tv.js-master/public/remote/ressources/images'
    at Object.fs.statSync (fs.js:684:18)
    at Object.exports.copyDirSyncRecursive (/usr/local/share/npm/lib/node_modules/yapp/node_modules/wrench/lib/wrench.js:179:23)
    at /usr/local/share/npm/lib/node_modules/yapp/lib/builder.js:124:16
    at Function._.each._.forEach (/usr/local/share/npm/lib/node_modules/yapp/node_modules/underscore/underscore.js:86:24)
    at Builder.genStatic (/usr/local/share/npm/lib/node_modules/yapp/lib/builder.js:122:7)
    at Builder.build (/usr/local/share/npm/lib/node_modules/yapp/lib/builder.js:59:10)
    at Application.build (/usr/local/share/npm/lib/node_modules/yapp/lib/app.js:47:39)
    at Command.<anonymous> (/usr/local/share/npm/lib/node_modules/yapp/bin/yapp.js:17:24)
    at Command.<anonymous> (/usr/local/share/npm/lib/node_modules/yapp/node_modules/commander/index.js:254:8)
    at Command.EventEmitter.emit (events.js:98:17)
```

**Comment by herrsebi** [comment]
For a quick workaround, just mkdir public/remote/ressources/images

It will work either way...
On 8. Juli 2013 at 21:49:10, Aristides Lourenco (notifications@github.com) wrote:

I got the same issue:

Error: ENOENT, no such file or directory '/Users/user/Downloads/tv.js-master/public/remote/ressources/images'
    at Object.fs.statSync (fs.js:684:18)
    at Object.exports.copyDirSyncRecursive (/usr/local/share/npm/lib/node_modules/yapp/node_modules/wrench/lib/wrench.js:179:23)
    at /usr/local/share/npm/lib/node_modules/yapp/lib/builder.js:124:16
    at Function._.each._.forEach (/usr/local/share/npm/lib/node_modules/yapp/node_modules/underscore/underscore.js:86:24)
    at Builder.genStatic (/usr/local/share/npm/lib/node_modules/yapp/lib/builder.js:122:7)
    at Builder.build (/usr/local/share/npm/lib/node_modules/yapp/lib/builder.js:59:10)
    at Application.build (/usr/local/share/npm/lib/node_modules/yapp/lib/app.js:47:39)
    at Command.<anonymous> (/usr/local/share/npm/lib/node_modules/yapp/bin/yapp.js:17:24)
    at Command.<anonymous> (/usr/local/share/npm/lib/node_modules/yapp/node_modules/commander/index.js:254:8)
    at Command.EventEmitter.emit (events.js:98:17)

**Comment by AaronO** [comment]
This is a not fatal error. Please refer to issue #10

**Comment by AaronO** [comment]
Fixed with 74d2dff654541e8d7a394cd3c4623d1604e0c431
